doc/nrf/ug_thingy53.rst

```diff
@@ -337,7 +337,6 @@ External flash
 During a FOTA update, there might not be enough space available in internal flash storage to store the existing application and network core images as well as the incoming images, so the incoming images must be stored in external flash storage.
```

Comment:
@de-nordic, please align this one. That's the last requested change.

Reply:
Maybe something like this:
```
Both the MCUboot bootloader and the application need a flash controller overlay.
For the MCUboot bootloader this is needed for the network core image update, and the overlay is automatically applied.
For the application this is needed to prevent bus faults when MCUmanager tries to access the simulated partition that contains the network core image data.
See :file:`thingy53_nrf5340_cpuapp.overlay` in a compatible sample for how this is done for the application.
```

--->

```
The MCUboot bootloader needs a flash controller overlay for the network core image update.
The overlay is applied automatically.
```

Reply:
I do not get the alignment request: https://github.com/nrfconnect/sdk-nrf/pull/7030/files#r897905065, but did the https://github.com/nrfconnect/sdk-nrf/pull/7030/files#r897907133.
Can you give me more info on what you mean by alignment?

Reply:
Hi @de-nordic
The alignment comment refereed to: https://github.com/nrfconnect/sdk-nrf/pull/7030/files#r894511803 (I had these messages next to each other on Github). If you applied second suggestion, then it's fixed already :)